ci/matlab.yml

```diff
@@ -0,0 +1,57 @@
+# This is a basic workflow to help you get started with MATLAB actions
```

Comment:
In the next paragraph, you terminate sentences with a period. Please do the same for these opening sentences.